sygnal/apnspushkin.py

```diff
@@ -278,7 +278,7 @@ async def dispatch_notification(self, n, device, context):
                             retry_delay, twisted_reactor=self.sygnal.reactor
                         )
 
-    def _get_payload_event_id_only(self, n):
+    def _get_payload_event_id_only(self, n, device):
```

Comment:
`device` needs an entry in the docstring, please (preferably one that says what type it is)

Reply:
updated